Package.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.2
```

Comment:
As far as I understand, `5.2` will make it unavailable for packages that use an earlier version of Swift, won't it?

```suggestion
// swift-tools-version:4.2
```